lib/reporters/base.js

```diff
@@ -1,23 +1,43 @@
-var util = require('util')
+'use strict'
 
-var constants = require('../constants')
-var helper = require('../helper')
+const util = require('util')
 
-var BaseReporter = function (formatError, reportSlow, useColors, browserConsoleLogOptions, adapter) {
+const constants = require('../constants')
+const helper = require('../helper')
+
+const BaseReporter = function (formatError, reportSlow, useColors, browserConsoleLogOptions, adapter) {
   this.adapters = [adapter || process.stdout.write.bind(process.stdout)]
 
-  this.onRunStart = function () {
+  this.USE_COLORS = false
+  this.EXCLUSIVELY_USE_COLORS = undefined
+  this.LOG_SINGLE_BROWSER = '%s: %s\n'
+  this.LOG_MULTI_BROWSER = '%s %s: %s\n'
```

Comment:
Seems like `LOG_SINGLE_BROWSER` and `LOG_MULTI_BROWSER` variables are redundant. I think:

    this.writeCommonMsg(util.format(this.LOG_SINGLE_BROWSER, type, log)) 
can be replaced with:

    this.writeCommonMsg(`${type}: ${log}\n`) 

This note may also refer to other variables, what do you think ?

Reply:
Yeah, these variables are base values, which can be redefined by the concrete reporters, so they should be kept. Also as other reporters in a wild may rely on these values removing them may introduce a breaking change.